src/app.module.ts

```diff
@@ -13,17 +14,22 @@ import { MacroComponent } from './components/macro';
 import { LegacyLoaderComponent } from './components/legacy-loader';
 import { NotificationComponent } from './components/notification';
 import { SvgKeystrokeKeyComponent, SvgOneLineTextKeyComponent, SvgTwoLineTextKeyComponent } from './components/svg/keys';
+import { ContenteditableModel } from './components/directives/contenteditable.component';
+import { MacroActionEditorComponent } from './components/macro/macro-action-editor/macro-action-editor.component';
 
 @NgModule({
     declarations: [
+        FORM_DIRECTIVES,
```

Comment:
I think we should rather use `FormsModule` according to [this](https://angular.io/docs/ts/latest/guide/forms.html).